### ISSUE TITLE
How to Disable/bypass checkstyle in gradle  via ./gradlew sonarqube

### ISSUE THREAD

**Issue by Sadashiv** [checkstyle/checkstyle#10975]
I have read check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

sonarqube version: 8.7
checkstyle plugin in sonarqube: 4.29

How to skip/bypass checkstyle during ./gradlew  sonarqube.
As of now i am not interested to go with checkstyle which is failing code anyalysis part. 

```bash
/var/tmp $ javac YOUR_FILE.javajavac src/main/java/org/resultsbridge/export/CsvDataExporter.java

src/main/java/org/resultsbridge/export/CsvDataExporter.java:589: error: : expected
            createLabelsLegend(startOfNextLegend + 1); // not to base on order
                                                     ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:591: error: : expected
            createResultStatusLegend(startOfNextLegend + 1);
                                                           ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:592: error: : expected
        case ROUGH_BIN_LEGEND -> startOfNextLegend = createRoughBinLegend(startOfNextLegend + 1);
                                                                                                ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:594: error: : expected
            createGroupLookUpLegend(startOfNextLegend + 1);
                                                          ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:596: error: : expected
            createArithmeticLegend(startOfNextLegend + 1);
                                                         ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:597: error: : expected
        case CARE_AREA_LEGEND -> startOfNextLegend = createCareAreaLegend(startOfNextLegend + 1);
                                                                                                ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:598: error: : expected
        case IMAGING_LEGEND -> startOfNextLegend = createImagingLegend(startOfNextLegend + 1);
                                                                                             ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:599: error: : expected
        default -> startOfNextLegend++;
               ^
src/main/java/org/resultsbridge/export//CsvDataExporter.java:599: error: illegal start of expression
        default -> startOfNextLegend++;
                ^
9 errors

/var/tmp $ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
  <!--
      If you set the basedir property below, then all reported file
      names will be relative to the specified directory. See
      https://checkstyle.org/config.html#Checker

      <property name="basedir" value="${basedir}"/>
  -->

  <metadata name="org.checkstyle.principle" value="Practice What You Preach"/>
  <metadata name="org.checkstyle.principle.description"
            value="In our config we should use all Checks that Checkstyle has"/>

  <property name="cacheFile" value="${checkstyle.cache.file}"/>

  <property name="severity" value="ignore"/>

  <property name="fileExtensions" value="java, properties, xml, vm, g, g4, dtd"/>

  <!-- BeforeExecutionFileFilters is required for sources that are based on java9 -->
  <module name="BeforeExecutionExclusionFileFilter">
    <property name="fileNamePattern" value="module\-info\.java$" />
  </module>

  <!-- Filters -->
  <module name="SeverityMatchFilter">
    <!-- report all violations except ignore -->
    <property name="severity" value="ignore"/>
    <property name="acceptOnMatch" value="false"/>
  </module>
  <module name="SuppressionFilter">
    <property name="file" value="${checkstyle.suppressions.file}"/>
  </module>
  <!-- Tone down the checking for test code -->
  <module name="SuppressionSingleFilter">
    <property name="checks" value="JavadocPackage"/>
    <property name="files" value=".*[\\/]src[\\/](test|it)[\\/]"/>
  </module>


/var/tmp $ cat YOUR_FILE.java
#[[PLACE YOU OUTPUT HERE]]

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
#[[PLACE YOUR OUTPUT HERE]]
```

How to skip/bypass checkstyle during ./gradlew  sonarqube.
As of now i am not interested to go with checkstyle which is failing code anyalysis part. 

Still not clear ???
see example - https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F

ATTENTION: FAILURE TO FOLLOW THE ABOVE TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED.

**Comment by romani** [comment]
Ask this question in stackoverflow or gradle support.
